UnityProject/Assets/Scripts/Items/Implants/Organs/Lungs.cs

```diff
@@ -167,6 +167,36 @@ private bool BreatheIn(GasMix breathGasMix, ReagentMix blood, float efficiency)
 				//TODO: Add pressureSafeMax check here, for hyperoxia
 			}
 		}
+
+
+		//If Styrene is present past a tenth of a mole you cant breath at all. The toxins shut down your lungs.
+		if (breathGasMix.GetMoles(Gas.Styrene) > 0.10)
```

Comment:
keep in mind that this will affect all lungs. I think we need a flexible way to handle toxins and their effects for particular lifeforms, so plasma is toxic for humans but not for plasmamen.